cluster/examples/kubernetes/nfs/crd.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: export.nfs.rook.io
```

Comment:
name should be plural: `exports.nfs.rook.io`

Reply:
i'm wondering if export is the right CRD type name though.  What's the correct terminology in the NFS world?  A NFS server instance has multiple exports, is that the right wording? This CRD is really about defining the NFS server and all its exports, so maybe the name should be `nfsserver.nfs.rook.io` instead? that may appease @travisn too since it uses a non-generic Kind name. thoughts?

after reading more, it looks like a CRD can have multiple servers also.  would the operator create multiple NFS server deployments/pods in that case in response to the single CRD being created by the user?  I would think that 1 CRD would only capture 1 server instance.  Let's review and agree on the cardinality we want, is the following correct?
1 NFS server CRD contains 1 NFS server
1 NFS server contains multiple NFS exports
1 NFS export contains 1 path or volume

That cardinality does not match the golang types.go code below (types.go says 1 export has multiple servers, which doesn't make sense to me), so either I have it wrong here or that code is wrong, so it would be good to all get on the same page.

Reply:
`nfsserver.nfs.rook.io` seems to be more appropriate in this case. 
The cardinality you mentioned is the one we want.
I named `export.nfs.rook.io` keeping other Rook CRD names in mind. 
But after reading your comment I realized that types.go doesn't make sense.
So types.go according to the bellow will be more appropriate. 
```yaml
group: nfs.rook.io
  names:
    kind: NFSServer
    listKind: NFSServerList
```
What do you suggest?

Reply:
I think the `NFSServer` kind better describes what the CRD is capturing and the operator will deploy.  That also addresses @travisn concerns about generic/conflicting names for the kind (e.g. if it was just Server that would be too generic).  Thank you for confirming the cardinality here, that makes a lot more sense to me now.